NEWS

```diff
@@ -1,3 +1,31 @@
+11-May-2022 Ignition 2.14.0
+
+    Features:
+
+    - Support KubeVirt platform
+    - Support AWS arn: URLs for S3 objects and access points
+    - Support reading configs from Azure IMDS "user data"
+    - Support S3 fetch via IPv6
+    - Add ignition-apply entrypoint to apply an Ignition config in a container
+
+  Changes:
+
+    - Delete userdata after provisioning on VirtualBox and VMware by default
+      (see operator notes for details)
+    - Support setting setuid/setgid/sticky mode bits (3.4.0-exp)
+    - Warn if setuid/setgid/sticky mode bits specified (3.0.0 - 3.3.0)
+    - Support UEFI Secure Boot on VMware
+    - Add arm64 support to ignition-validate container
+    - Document S3 fetch semantics in operator notes
+
+  Bug Fixes:
+
+    - Fix disabling systemd units with pre-existing enablement symlinks
+    - Fix reuse of statically keyed LUKS volume (2.12.0 regression)
```

Comment:
Tiny nit: `volumes` for consistency with the pluralization of the other items.

Reply:
Done!